frontend/app/src/components/history/events/HistoryEventsView.vue

```diff
@@ -784,6 +815,14 @@ const includeOnlineEvents: ComputedRef<boolean> = useEmptyOrSome(
       />
 
       <TransactionFormDialog :loading="sectionLoading" />
+
+      <MissingRulesDialog
+        v-model="missingRulesDialog"
```

Comment:
maybe do `v-if` here? how much of this renders when the dialog is not visible?

Reply:
it renders the content, will is if to keep it off the dom unless needed